docs/guides/protocol-levers.md

```diff
@@ -11,125 +11,121 @@ All existing levers are listed below, grouped by the contract.
 
 The following contracts are upgradeable by the DAO voting:
 
+- [`LidoLocator`](/contracts/lido-locator)
 - [`Lido`](/contracts/lido)
+- [`StakingRouter`](/contracts/staking-router)
 - [`NodeOperatorsRegistry`](/contracts/node-operators-registry)
-- [`LidoOracle`](/contracts/lido-oracle)
+- [`AccountingOracle`](/contracts/accounting-oracle)
+- [`ValidatorsExitBusOracle`](/contracts/validators-exit-bus-oracle)
+- [`WithdrawalVault`](/contracts/withdrawal-vault)
+- [`WithdrawalQueueERC721`](/contracts/withdrawal-queue-erc721)
+- [`LegacyOracle`](/contracts/legacy-oracle)
 
-Upgradeability is implemented by the Aragon kernel and base contracts. To upgrade an app, one needs
-the `dao.APP_MANAGER_ROLE` permission provided by Aragon. All upgradeable contracts use the
-[Unstructured Storage pattern] in order to provide stable storage structure across upgrades.
+Upgradeability is implemented either by the Aragon kernel and base contracts OR by the [OssifiableProxy](/contracts/ossifiable-proxy) instances.
```

Comment:
WithdrawalVault is not behind the OssifiableProxy AFAIK

Reply:
It's also an ossifiable proxy, though, the source code is different, it's correct

Reply:
Pushed a more accurate version, thx!